app.js

```diff
@@ -54,11 +54,12 @@ async function formSubmit(event) {
         String.fromCharCode(...new Uint8Array(hashBuffer))
       );
       const integrityMetadata = `sha384-${base64string}`;
-      const scriptEl = `<script src="${encodeURI(
+      const scriptEl = `<span style="color: #ffa07a">&lt;script src=</span><span style="color:#abe338">&quot;${encodeURI(
         url
-      )}" integrity="${integrityMetadata}" crossorigin="anonymous"></script>`;
+      )}&quot;</span> <span style="color: #ffa07a">integrity=</span><span style="color:#abe338">&quot;${integrityMetadata}&quot;</span> <span style="color: #ffa07a">crossorigin=</span><span style="color:#abe338">&quot;anonymous&quot;</span><span style="color: #ffa07a">&gt;&lt;/script&gt;</span>`;
 
-      resultDiv.innerText = scriptEl;
+      resultDiv.classList.add("is-active");
```

Comment:
I think you need to remove the class in `resetInterface`, otherwise the following issue comes up:
- get the hash for a working url
- get the hash for a url that doesnt work
- you'll see the padding above the error text.

Reply:
fixed :)